### ISSUE TITLE
Provide details about a post when embeding a link in Facebook

### ISSUE THREAD

**Issue by brianherbert** [ushahidi/crowdmap-issues#164]
I think we just need to tweak the meta tags with post data, right?

![screen shot 2013-05-13 at 11 05 57 am](https://f.cloud.github.com/assets/106068/493856/ca823db4-bb71-11e2-9637-c15d556c2f60.png)

**Comment by brianherbert** [comment]
Figured out the problem here. Facebook is choking on the lack of a gatekeeper cookie when they return to Crowdmap to grab the meta data they need. We need to find a way to route them properly without making the cookie necessary (I know, easier said than done).

**Comment by brianherbert** [comment]
Spent the past couple hours trying to figure all this out. As a side effect, I much better understand our gatekeeper process.

https://developers.facebook.com/tools/debug/og/object?q=https%3A%2F%2Fcrowdmap.com%2Fpost%2F832 shows why Facebook links are not pulling the correct data in Facebook. Apparently there is some funny redirecting going on that drops Facebook on the homepage.

I tried disabling my cookies and setting my user agent to the Facebook UA with no success. One time I landed on the homepage when I visited https://crowdmap.com/post/832/ but other than that one time, it seems to load just fine.

I checked my HTTP headers and I'm not seeing the 302 redirects that the Facebook debugger is reporting.

I did add some additional location checks that bypass the gatekeeper ("/assets" "/user" "/post') so oembed support now works fine.
